modifs partie bio intro

LeilaRenard opened the pull request:
est-ce que j'ai bien fait la pull request comme il faut?

LeilaRenard committed:
modifs partie bio intro

(ligne 193 j'ai pas compris pourquoi A=C et non pas A=T)

LeilaRenard committed:
annulation "envisager" retour phrase d'avant

n-peugnet commented on pdf/doc.tex:
```diff
@@ -168,23 +168,25 @@ \subsection{Fonctionnalités et caractéristiques}
-et un certain nombre de démonstrations de faisabilité du stockage sur l’\ac{adn} ont été réalisées lors des dernières années.
-Les travaux publiés pour l’instant se basent essentiellement sur l’utilisation d’\emph{oligonucléotides} qui sont des courts segments d’\ac{adn}.
+Il est donc possible d'établir un parallèle entre l'information génétique, codée par les 4 nucléotides formant l'\ac{adn}, en base~4, et la donnée informatique, codée par une suite de bits en base~2.
+Il est donc naturel de penser à utiliser l'\ac{adn} pour stocker des données.
```
haha oups en fait je ne tenais pas particulièrement à cette phrase. Et maintenant ça fait deux fois "il est donc" à la suite.
Mais bon en fait je vais plutôt te laisser finir et reviewer seulement quand tu auras fini. Écris comme tu le sens, j’arrête de t'embêter.

LeilaRenard committed:
micro erreur de frappe + suggestions

sinon : (pour éviter un nouveau pull request)
- l.623 dans ton dernier commit (pas visible dans ma branche) : d'y -> s'y
- l.340 quand tu dit "sans même parler de leur prix", peut être mettre une estimation ? sinon ça fait un peu vague à mon sens
- l.267 limiter la casse c'est marrant mais jsp si c'est très rapport de stage haha

n-peugnet commented on pdf/doc.tex:
```diff
@@ -168,23 +168,24 @@ \subsection{Fonctionnalités et caractéristiques}
 Dans ses travaux, Church souhaite pouvoir contrôler le taux de GC et limiter les répétitions d’un même nucléotide.
 Le taux de GC est la proportion de nucléotides G et C dans une séquence donnée. 
-Les appariements GC ont trois liaisons hydrogène tandis que les appariements AT n'en ont que deux.
+Les appariements GC sont formées par trois liaisons hydrogène tandis que les appariements AT n'en ont que deux, les rendant moins stables.
```
Hmm ptêtre un peu chelou de garder n'en ont que deux du coup ? Mais bon allez ça part !

n-peugnet reviewed (approved):
Super, merci beaucoup !
